t/rose-date/00-basic.t

```diff
@@ -317,7 +293,7 @@ __OUT__
 TEST_KEY=$TEST_KEY_BASE-as-total-PT1S-m
 run_pass "$TEST_KEY" rose date --as-total=m PT1S
 file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<'__OUT__'
-0.0166666666667
+0.016666666666666666
```

Comment:
This test is always going to be flaky, could grep for `^0.01666` instead.

Reply:
I quite like leaving it sensitively dependent/flaky but I take your point. I wouldn't worry too much since I delete it in the second PR because it duplicates functionality that should be in isodatetime's testing.